internal/cmd/options.go

```diff
@@ -15,3 +19,24 @@ func (opts *SignerFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
 	SetPflagKey(fs, &opts.Key)
 	SetPflagSignatureFormat(fs, &opts.SignatureFormat)
 }
+
+// Common option struct.
+type Common struct {
+	Debug bool
+}
+
+// ApplyFlags applies flags to a command flag set.
+func (opts *Common) ApplyFlags(fs *pflag.FlagSet) {
+	fs.BoolVarP(&opts.Debug, "debug", "d", false, "debug mode")
+}
+
+// SetLoggerLevel sets up the logger based on common options.
+func (opts *Common) SetLoggerLevel() (context.Context, logrus.FieldLogger) {
+	var logLevel logrus.Level
+	if opts.Debug {
+		logLevel = logrus.DebugLevel
+	} else {
+		logLevel = logrus.WarnLevel
+	}
```

Comment:
Got a minor question here, looks like the info level will never be used by Notation CLI? logrus has a default of InfoLevel, i.e. info level and anything above. (https://pkg.go.dev/github.com/sirupsen/logrus#readme-level-logging).

Reply:
Debug level also includes info level logs. Let's keep it simple here and add it when we need it. @yizha1 was agreed with this way.

Reply:
Agreed on this result:
- No debug flag: Warning + error
- with debug flag: debug + info + warning +error
- (future) with verbose: info+ warning + error